wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -277,19 +285,27 @@ func (s *WSBroadcastServer) StartWithHeader(ctx context.Context, header ws.Hands
 						ws.RejectionReason(fmt.Sprintf("Missing HTTP header %s", HTTPHeaderFeedClientVersion)),
 					)
 				}
+				if connectingIP == nil {
+					if addr, ok := conn.RemoteAddr().(*net.TCPAddr); ok {
+						connectingIP = addr.IP
+					}
+				}
+
+				if config.ConnectionLimits.Enable && !s.clientManager.connectionLimiter.IsAllowed(connectingIP) {
+					return nil, ws.RejectConnectionError(
+						ws.RejectionStatus(http.StatusTooManyRequests),
+						ws.RejectionReason("Too many open websocket connections."),
```

Comment:
This is a nitpick, but maybe specifically note that there's too many feed connections vs other types of websocket connections

Reply:
Changed to say feed.